README.md

```diff
@@ -95,6 +97,17 @@ If you use `SpecialDay` instances, you need to install the `jeremeamia/superclos
 $ composer require jeremeamia/superclosure
 ```
 
+#### JSON serialization
+
+All classes can be encoded to JSON
+
+```php
+$json = json_encode($business);
+```
+
+The `SpecialDay` instances require a context in order extract the data from the callable.
+This is automatically set to `new \DateTime('now', new \DateTimeZone('UTC'))` for `json_encode()` call only.
```

Comment:
Could we use something like this instead ? https://github.com/dragosprotung/business/blob/master/src/Business.php#L44